protocol/backend/backend_test.go

```diff
@@ -14,19 +15,19 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-const defaultXMRTakerAddress = "0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1"
```

Comment:
I updated the code below to derive this address. My thinking was that it was better to have all the Ganache key related values defined in one place.